docs/DeveloperGuide.md

```diff
@@ -659,6 +659,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Edit a client's details
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to view list of all clients.
+1. FitBook shows a list of clients.
+1. User requests to edit the details of a specific client in the list.
```

Comment:
Should we add "edit by index here", would flow better with the 3a extension too.

Reply:
i think no need might confuse the user